main_init_test.go

```diff
@@ -150,7 +151,17 @@ func TestInitWithProvisioners(t *testing.T) {
 	provs, err := loader.LoadProvisionersFromDirectory(filepath.Join(td, ".score-k8s"), loader.DefaultSuffix)
 	assert.NoError(t, err)
 	if assert.Greater(t, len(provs), 2) {
-		assert.Equal(t, "template://two", provs[0].Uri())
-		assert.Equal(t, "template://one", provs[1].Uri())
+		expectedProvisioners := map[string]bool{
```

Comment:
@7h3-3mp7y-m4n sorry for being annoying, what do you think about this way to check that the expected provisioners are there without checking the order but leveraging `slices` functions:
```go
expectedProvisionerUris := []string{"template://one", "template://two"}
for _, expectedUri := range expectedProvisionerUris {
	assert.True(t, slices.ContainsFunc(provs, func(p provisioners.Provisioner) bool {
		return p.Uri() == expectedUri
	}), fmt.Sprintf("Expected provisioner '%s' not found", expectedUri))
}
```

?

Reply:
No no @delca85  you are not annoying, TBH I like your approach, I wrote it way cause I was debugging at the same time. I added a map because of its fast lookup  ... As we are dealing with a small number of inputs cause it's a test we can definitely use your approach, and it's more flexible and has fewer lines .. Thanks!

Reply:
Cool, thank you @7h3-3mp7y-m4n !